Jenkinsfile

```diff
@@ -1081,7 +1082,7 @@ pipeline {
                             rpm_test_post(env.STAGE_NAME, env.NODELIST)
                         }
                     }
-                } // stage('Test CentOS 7 RPMs')
+                } // stage('Test RPMs on EL 8.6')
                 stage('Test RPMs on Leap 15.4') {
```

Comment:
Do you still want to be testing on Leap 15.4 here, or should this be 15.5 with this move to 15.6?

Reply:
If the goal here is to run this test on the N-1 version, then I should.  Is that the intention of this stage?

Reply:
Generally yes-ish.  The ultimate goal is to run this stage on any other version of a distro that is supported for a release.  Generally that is only `n-1` if even that.  For Leap, I think we tend to be less willing to support `n` and `n-1` but for EL, we do tend to be more willing to do that.

Which versions we support in a release is supposed to be tracked in SRS tickets such as [this one](https://daosio.atlassian.net/browse/SRS-416) for the _2.8 Community Release_.  That ticket unfortunately seems to be incomplete at this time.

The [SRS ticket](https://daosio.atlassian.net/browse/SRS-407) for the _2.6 Community Release_ though is a more complete example.  You can see that it only specifies Leap 15.5.  That would/should be why that stage is generally skipped such as you can see [here](https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-15561/2/pipeline/).  That should be driven by the value of the `CI_RPMS_leap15.4_TEST` parameter in the `Jenkinsfile` but I see that that parameter is actually currently missing.

Similarly, the _Test RPMs on EL 8.6_ stage (it should actually be 8.8 with main functional testing on 8.10 -- clearly things have slipped in that regard) is being run because SRS-407 says we are covering two releases for EL8.

I am at a loss however to explain why that _Test RPMs on EL 8.6_ stage is being run as the `Jenkinsfile` is also missing the `CI_RPMS_el8.6_TEST` that the stage skip code for that stage wants: https://github.com/daos-stack/pipeline-lib/blob/e6831a2ec6c8b9ee4ada79cd50aa91332fd98993/vars/skipStage.groovy#L489-L501.

In any case, that's how it's all _supposed_ to work.